src/test/java/name/valery1707/junit/rule/ConditionalIgnoreRuleTest.java

```diff
@@ -312,4 +312,54 @@ public void testNonStaticCorrect() {
 			)
 		;
 	}
+
+	public static class RepeatableTest {
+		@Rule
```

Comment:
\[Checkstyle\] WARNING: 'member def modifier' has incorrect indentation level 16, expected level should be 4\.